.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+on:
+  push:
+    tags-ignore:
```

Comment:
so for `tags-ignore`, would this mean that it does or does trigger this action when we push tags to main?